### PR TITLE
feat(spider): add Coop Alleanza 3.0 (IT, 374)

### LINKED ISSUE #11410: Coop Alleanza 3.0
### Brand name

Coop Alleanza 3.0

### Wikidata ID

Q21189584

### Store finder url(s)

[Webpage](https://www.coopalleanza3-0.it/fare-spesa/elenco-negozi.html)
[JSON (to be POSTed without body)](https://www.coopalleanza3-0.it/storeLocatorServlet/?operation=getStores)

### Sample store page url

https://www.coopalleanza3-0.it/fare-spesa/elenco-negozi/dettaglio-negozio/2542-venezia---piazzale-roma.html

### Countries?

IT

### Difficulty?

Medium - some customisation or data cleansing

### Number of POI?

100+

### Behaviours

- [X] Has coordinates?
- [X] Has opening hours?
- [ ] JSON API?
- [ ] Embedded JSON?
- [X] HTML markup?
- [ ] Structured Data? (`pipenv run scrapy sd (specific page url)`) or [validator](https://validator.schema.org/) has content
- [ ] Open Graph Data?
- [ ] Sitemap? (`pipenv run scrapy sitemap (url)`)
- [ ] Requires proxy?
- [ ] Has 'bot protection' (Cloudflare, etc)?

### PR DESCRIPTION
**Pull request by McCio**
I also made possible to parse the "closed" status in opening hours, with localisation support like the other vars

Fixes #11410

### CODE REVIEW

**Comment by McCio** [comment]
As a note, it results in 58 items as it needs to crawl the webpage for each - in total they are 374

File: locations/hours.py [hunk 1]
```diff
@@ -1093,6 +1134,53 @@ def hours_extraction_regex(
         )
         return full_regex
 
+    @staticmethod
+    def closed_days_extraction_regex(
```
Comment (iandees) [inline locations/hours.py:1138]: I'd like to remove this code duplication here. Can you find a better way to do this?

File: locations/hours.py [hunk 2]
```diff
@@ -1234,6 +1329,14 @@ def extract_hours_from_string(
                     time_end_24h = time.strptime(time_end, "%I:%M%p")
                     time_end_24h = time.strftime("%H:%M", time_end_24h)
                     results.append((days_in_range, time_start_24h, time_end_24h))
+        if len(results_closed) > 0:
```
Comment (iandees) [inline locations/hours.py:1332]: Please add tests to cover the situations you're adding here.

Comment (McCio) [reply locations/hours.py]: I'll split this on a different PR to avoid confusion and make cleaner code

Comment (McCio) [reply locations/hours.py]: @iandees done that in #11469

**Pull request merged**